tests/php/Security/InheritedPermissionsFlusherTest.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace SilverStripe\Security\Tests;
+
+use Psr\SimpleCache\CacheInterface;
+use SilverStripe\Core\Cache\CacheFactory;
+use SilverStripe\Dev\SapphireTest;
+use SilverStripe\Security\InheritedPermissionFlusher;
+use SilverStripe\Security\Member;
+use SilverStripe\Security\Group;
+use SilverStripe\Core\Injector\Injector;
+use SilverStripe\Security\Tests\InheritedPermissionsFlusherTest\TestCacheFlusher;
+use SilverStripe\Core\Config\Config;
+
+class InheritedPermissionsFlusherTest extends SapphireTest
+{
+    protected static $fixture_file = 'InheritedPermissionsFlusherTest.yml';
+
+    public function setUp()
```

Comment:
`setUp()` is protected nowadays

Reply:
Good catch. I'm really surprised this didn't throw a strict warning.